### PR TITLE
All Adventure-Cards completly translated.

### PR DESCRIPTION
**Pull request by idlaviV**
I hope I got the tokens right. There might be inconsistencies with white-spaces in extra-texts. This replaces my pull request from my forked repository.

### CODE REVIEW

File: card_db/de/cards_de.json [hunk 1]
```diff
@@ -1,340 +1,293 @@
...
         "description": "+1 Aktion<br>+1 Coin<br>Nimm dir pro Karte, die dein rechter Mitspieler in seinem letzten Zug genommen hat, ein Silber. Ist diese Karte im Spiel, darfst du sie gegen einen Krieger eintauschen anstatt sie abzulegen. (Diese Karte gehört nicht zum Vorrat.)", 
         "extra": "Einen SCHATZSUCHER erhältst du nur, wenn du einen PAGEN eintauschst. Der SCHATZSUCHER ist ein Reisender, der im Spielverlauf in einen KRIEGER eingetauscht werden kann. Wenn du den SCHATZSUCHER ausspielst, erhältst du +1 Aktion sowie +1 Coins. Für jede Karte, die dein rechter Mitspieler in seinem letzten Zug gekauft oder genommen hat, nimmst du ein SILBER vom Vorrat und legst es ab. In der Aufräumphase darfst du entscheiden, ob du den SCHATZSUCHER ablegst oder zurück auf den entsprechenden Stapel legst. Wenn du das tust, erhältst du einen KRIEGER und legst ihn ab. Die angegebenen Kosten werden nicht bezahlt.", 
         "name": "Schatzsucher"
     },
     "Treasure Trove": {
-        "description": "2 <*COIN*><n>When you play this, gain a Gold and a Copper.", 
+        "description": "2 <*COIN*><n>WWenn du diese Karte ausspielst, nimm dir ein Gold und ein Kupfer.",
```
Comment (sumpfork) [inline card_db/de/cards_de.json]: WWenn -> Wenn

File: card_db/de/cards_de.json [hunk 2]
```diff
@@ -1,340 +1,293 @@
...
         "name": "Krieger"
     },
     "Wine Merchant": {
-        "description": "+1 Buy<br>+4 Coins<n>Put this on your Tavern mat.<line>At the end of your Buy phase, if you have at least 2 Coins unspent, you may discard this from your Tavern mat.", 
+        "description": "+1 Kauf<br>+4 Coins<n>Lege diese Karte auf dein Wirtshaustableau.<line>Wenn du am Ende deiner Kaufphase mindestens 2 Coins nicht ausgegeben hast, darfst d diese Karte von deinem Tableau ablegen.",
```
Comment (sumpfork) [inline card_db/de/cards_de.json]: d -> du

File: card_db/de/cards_de.json [hunk 3]
```diff
@@ -1,340 +1,293 @@
...
-        "untranslated": "description, extra"
+        "description": "Lege deinen +1 Coin-Marker auf einen Aktions-Vorratsstapel.<n> (Wenn du eine Karte von diesem Stapel ausspielst, erhälst du zuerst +1 coin.)", 
+        "extra": "Lege deinen +1 Coin-Marker auf einen beliebigen Aktions-Vorratsstapel. Solange dieser Marker auf dem Stapel liegt, erhält du +1 coin, sobald du eine Karte, die von diesem Stapel stammt, ausspielst. <n> Durch den Erwerb eines weiteren TRAININGS oder das Aufrufen eines LEHRERS darfst du den Marker auf einen anderen Aktions-Vorratsstapel legen.", 
+        "name": "Training"
     },
     "Transmogrify": {
-        "description": "+1 Action<n>Put this on your Tavern mat.<line>At the start of your turn, you may call this, to trash a card from your hand, gain a card costing up to 1 Coin more than it, and put that card into your hand.", 
+        "description": "+1 Aktion <n> Lege diese Karte auf dein Wirtshaustableau. <line> Zu Beginn deines Zuges darfst du diese Karte von deinem Tableau aufrufen. Wenn du das tut: Entsorge eine Handkarte und nimm dir eine Karte, die bis zu 1 coin mehr kostet als die entsorgte Karte. Nimm diese Karte auf die Hand.",
```
Comment (sumpfork) [inline card_db/de/cards_de.json]: tut -> tust

File: card_db/de/cards_de.json [hunk 4]
```diff
@@ -1,340 +1,293 @@
...
         "extra": "Einen SOLDATEN erhältst du nur, wenn du einen KLEINBAUERN eintauschst. Der SOLDAT ist ein Reisender, der im Spielverlauf in einen FLÜCHTLING eingetauscht werden kann. Wenn du den SOLDATEN ausspielst, erhältst du +2 Coins sowie für jede andere Angriffskarte (außer diesem SOLDATEN), die du zu diesem Zeitpunkt im Spiel hast, +1 Coins. Außerdem muss jeder Mitspieler, der 4 oder mehr Karten auf der Hand hat, 1 Karte ablegen. In der Aufräumphase darfst du entscheiden, ob du den SOLDATEN ablegst oder zurück auf den entsprechenden Stapel legst. Wenn du das tust, erhältst du einen FLÜCHTLING und legst ihn ab. Die angegebenen Kosten werden nicht bezahlt.", 
         "name": "Soldat"
     },
     "Storyteller": {
-        "description": "+1 Action<br>+1 Coin<n>Play up to 3 Treasures from your hand. Pay all of your _ Coins. +1 Card per Coin paid.", 
+        "description": "+1 Aktion<br>+1 Coin<n>Spiele bis zu 3 Geldkarten aus deiner Hand aus. Zähle alls _ coins und du erhälst:<n> + 1 Kartepro gezahltem _ coins. <n> (Du hast jetzt 0 coins.)",
```
Comment (sumpfork) [inline card_db/de/cards_de.json]: alls -> alle und Kartepro -> Karte pro

File: card_db/de/cards_de.json [hunk 5]
```diff
@@ -1,340 +1,293 @@
...
-        "name": "Seeweg", 
-        "untranslated": "description, extra"
+        "description": "Nim dir eine Aktionskarte, die bis zu 4 coins kostet. Lege deinen + 1 Kauf-Marker auf diesen Vorratsstapel. (Wenn du eine Karte von diesem Stapel ausspielst, erhälst du zuerst +1 Kauf.)",
```
Comment (sumpfork) [inline card_db/de/cards_de.json]: Nim -> Nimm

File: card_db/de/cards_de.json [hunk 6]
```diff
@@ -1,340 +1,293 @@
...
-        "extra": "When you buy this you get +1 Buy (letting you buy another Event or a card afterwards). Then look at the top 5 cards of your deck, discarding 3 and putting the rest on top of your deck in any order. If there are fewer than 5 cards even after shuffling, you still discard 3 of them; if there are only 3 cards left between your deck and discard pile, all 3 will be discarded. Scouting Party is unaffected by the -1 Card token; if it is on top of your deck, replace it after resolving Scouting Party.", 
-        "name": "Spähtrupp", 
-        "untranslated": "description, extra"
+        "description": "+1 Kauf<n>Sieh dir die obersten 5 Karten deines Nachziehstapels an. Lege 3 davon ab und den Rest in beliebiger Reihenfolge zurück auf den Nachziehstapel.", 
+        "extra": "Du erhälst + 1 Kauf. Schau dir die obersten 5 Karten deines Nachziehstapels an. Lege 3 davon ab und de Rest in beliebiger Reihenfolge zurück auf den Nachziehstapel. Hast du nach dem Mischen des Ablagestapels weniger als 5 Karten zur Verfügung, legst du zuerst - wenn möglich - 3 Karten ab. Nur den rest (0,1 oder 2 Karten) legst du zurück auf den Nachziehstapel.",
```
Comment (sumpfork) [inline card_db/de/cards_de.json]: rest -> Rest

File: card_db/de/cards_de.json [hunk 7]
```diff
@@ -1,340 +1,293 @@
...
-        "untranslated": "description"
+        "name": "Königliche Kutsche"
     },
     "Save": {
-        "description": "+1 Buy<n>Once per turn: Set aside a card from your hand, and put it into your hand at end of turn (after drawing).", 
-        "extra": "You can only buy this once per turn. When you do, you get +1 Buy (letting you buy another Event or a card afterwards), set aside a card from your hand face down (the other players do not get to see it), and put it into your hand at the end of the turn, after drawing your hand for the next turn. For example you might set aside an unplayed Copper, and then after drawing your 5 cards for next turn, add the Copper to your hand.", 
-        "name": "Zuflucht", 
-        "untranslated": "description, extra"
+        "description": "+1 Kauf<n>Einmal pro Zug: Lege eine Handkarte zur Seite und nimm sie am Ende deines Zuges wieder auf die Hand (nach dem Ziehen).", 
+        "extra": "Dieses Ereignis darf nur 1 x pro Zug erworben werden. Du erhälst +1 Kauf und legst eine beliebige handkarte verdeckt zur Seite. Nachdem du in der Aufräumphase Karten nachgezogen hast, nimmst du die zur Seite gelegte Karte wieder auf die Hand.",
```
Comment (sumpfork) [inline card_db/de/cards_de.json]: handkarte -> Handkarte

File: card_db/de/cards_de.json [hunk 8]
```diff
@@ -1,340 +1,293 @@
...
+        "description": "+1 Karte<br>+1 Aktion<n>Lege diese Karte auf dein Wirtshaustableau.<line>Zu Beginn deines Zuges darfst du diese Karte von deinem Tableau aufrufen. Wenn du das tust, musst du eine Handkarte entsorgen.", 
         "extra": "Du erhältst +1 Karte und +1 Aktion. Lege diese Karte auf dein Wirtshaustableau. Du darfst diese Karte zu Beginn deines Zuges vom Tableau aufrufen. Wenn du das tust, lege die Karte in deinen Spielbereich. Dann musst du eine beliebige Karte aus deiner Hand entsorgen. Lege den RATTENFÄNGER in der Aufräumphase ab.", 
-        "name": "Rattenfänger", 
-        "untranslated": "description"
+        "name": "Rattenfänger"
     },
     "Raze": {
-        "description": "+1 Action<n>Trash this or a card from your hand. Look at the number of cards from the top of your deck equal to the cost in _ Coins of the trashed card. Put one into your hand and discard the rest.", 
+        "description": "+1 Aktion<n>Entsorge diese Karte oder eine Handkarte.<n> Schau die so viee Karten oben von deinem Nachziehstapel an, wie die entsorgte Karte X coins gekostet hat. Nimm eine davon auf die Hand und lege den Rest ab.",
```
Comment (sumpfork) [inline card_db/de/cards_de.json]: viee -> viele

**Comment by idlaviV** [comment]
Should be fine now.

**Comment by sumpfork** [comment]
Still a coupled of typos, but can correct later.

**Pull request merged**